public/app/features/templating/templateSrv.js

```diff
@@ -107,10 +111,15 @@ function (angular, _) {
       });
     };
 
-    this.getAllValue = function(variable) {
+    this.getAllValue = function(variable, format) {
       if (variable.allValue) {
         return variable.allValue;
       }
+
```

Comment:
what do you mean batched values?

> Line 38 is added exactly for that case.

Like I said think we need an option to disable allValue escaping, but disabling always will have issues, but not sure.